crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -566,7 +546,7 @@ impl AlgorithmUpdaterV1 {
         let projection_portion: u128 = self
             .unrecorded_blocks
             .iter()
-            .map(|(_, &bytes)| (bytes as u128))
+            .map(|(_, &bytes)| u128::from(bytes))
             .fold(0_u128, |acc, n| acc.saturating_add(n))
```

Comment:
This gets called semi-infrequently (only when we get a batch back from the committer). It could become more frequent I guess.

Yes. An alternative would be having the "total unrecorded bytes" tracked and when we remove blocks we subtract their bytes from that total. Then this would just be a multiplication of that. I can add that if you prefer.

Reply:
Done.